passgithelper.py

```diff
@@ -314,6 +314,13 @@ def get_value(
     ),
     "entry_name": EntryNameExtractor(option_suffix="_username"),
 }
+_password_extractors = {
+    _line_extractor_name: SpecificLineExtractor(0, 0, option_suffix="_password"),
+    "regex_search": RegexSearchExtractor(
+        r"^password: +(.*)$", option_suffix="_password"
+    ),
+    "entry_name": EntryNameExtractor(option_suffix="_password"),
```

Comment:
This extractor makes no sense here. The password would then be in the configuration file / password store folder structure and not encrypted at all.